.github/workflows/release_chart.yaml

```diff
@@ -20,14 +21,15 @@ jobs:
 
       - name: Install chart-releaser
         run: |
-          wget https://github.com/helm/chart-releaser/releases/download/v1.4.1/chart-releaser_1.4.1_linux_amd64.tar.gz
-          tar -zxf chart-releaser_1.4.1_linux_amd64.tar.gz cr
+          HELM_CHART_RELEASER_VERSION=$(curl -fsSL https://github.com/helm/chart-releaser/releases/latest -H "Accept: application/json" | jq -r .tag_name)
```

Comment:
how about https://github.com/helm/chart-releaser-action?

https://github.com/runatlantis/helm-charts/blob/main/.github/workflows/release.yaml  i think it will simplify flow